### ISSUE TITLE
Fix semantics of masked load/store

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/intel-intrinsics#109]
From spec and Intel doc, the masked load/store should trigger no segfault.

With AMD spec, it _could_ trigger a segfault.

Now, in reality, it's complicated:
https://erik.science/2019/06/21/AVX-fun.html
and Intel doesn't follow its own specification.

which means the instructions with masked load/store cannot be relied upon at all for accessing non-padded memory.
It is also a sidechannel attack potential attack!

This conveniently make the masked load/store much easier to implement for arm64.

**tl;dr** semantics of masked load/store should be more restricted than in Intel Guide

**Comment by p0nce** [comment]
Done
